uast/uast.go

```diff
@@ -405,6 +405,34 @@ const (
 	// (e.g. Visual Basic).
 	This
 
+	// List/Map/Set comprehensions are expressions that iterate over a container, optionally
+	// using restricting conditions, and generate a new List/Dict/Set with a subset of the
+	// elements of the original container optionally with some transformation applied to
+	// every element. These are parent nodes where the expression sublements will be mapped
+	// on the Comprehension* roles.
```

Comment:
This comment will probably be out of place in the auto-generated documentation as it is not just before any declaration.  Should be merged with other comments for it to work correctly.